sonar-project.properties

```
@@ -3,5 +3,6 @@ sonar.organization=moveit
 
 sonar.python.version=3
 sonar.sources=.work/target_ws/src
-
+sonar.cfamily.llvm-cov.reportPath=.work/target_ws/coverage.info
+sonar.scm.exclusions.disabled=true
```

This generated file seems to be particularly noisy.

```suggestion
sonar.scm.exclusions.disabled=true
sonar.exclusions=.work/target_ws/src/moveit2/moveit_planners/test_configs/prbt_ikfast_manipulator_plugin/src/prbt_manipulator_ikfast_solver.cpp
```